src/constants/links.js

```diff
@@ -18,6 +18,9 @@ export default {
   privacyPolicy: {
     to: '/privacy',
   },
+  dataProcessingAgreement: {
+    to: '/data-processing-agreement',
```

Comment:
```suggestion
    to: '/dpa',
```

Can we maybe make this shorter? :)